eidas_node/saml.py

```diff
@@ -0,0 +1,226 @@
+"""Conversion of SAML Requests/Responses and Light Requests/Responses."""
+from collections import OrderedDict
+from datetime import datetime
+from typing import Dict, Optional, Set, Type, TypeVar
+
+import xmlsec
+from lxml import etree
+from lxml.etree import Element, ElementTree, QName, SubElement
+
+from eidas_node.constants import StatusCode, SubStatusCode
+from eidas_node.errors import ValidationError
+from eidas_node.models import LevelOfAssurance, LightRequest, LightResponse, NameIdFormat, Status
+from eidas_node.utils import get_element_path
+
+NAMESPACES = OrderedDict([
```

Comment:
This can be a regular dict, or am I missing something?

Reply:
OrderedDict was necessary to produce identical XML documents in tests. I've switched to `assertXMLEqual`, so `NAMESPACES` was changed to a regular dict.